src/Writers/HTMLWriter.jl

```diff
@@ -1606,10 +1620,14 @@ mdconvert(b::Markdown.BlockQuote, parent; kwargs...) = Tag(:blockquote)(mdconver
 mdconvert(b::Markdown.Bold, parent; kwargs...) = Tag(:strong)(mdconvert(b.text, parent; kwargs...))
 
 function mdconvert(c::Markdown.Code, parent::MDBlockContext; kwargs...)
-    @tags pre code
+    @tags pre code button
     language = Utilities.codelang(c.language)
     language = isempty(language) ? "none" : language
-    pre(code[".language-$(language)"](c.code))
+    code_block = [
+        button[".copy-button .button .fas .fa-copy", Symbol("data-clipboard-text")=>c.code],
```

Comment:
Might be nice to have a `title` attribute on each button as well that would probably say "Copy to clipboard".